From 1064c1916c2b34607d2fa24be2bc22a802c64b7b Mon Sep 17 00:00:00 2001
From: Jeremy Drake <github@jdrake.com>
Date: Fri, 8 Jan 2021 10:31:36 -0800
Subject: [PATCH 02/11] cmake: bootstrap using only system libs built without
 cmake

---
 mingw-w64-cmake/PKGBUILD | 21 ++++++++++++---------
 1 file changed, 12 insertions(+), 9 deletions(-)

diff --git a/mingw-w64-cmake/PKGBUILD b/mingw-w64-cmake/PKGBUILD
index ac89a0118..481f2376a 100644
--- a/mingw-w64-cmake/PKGBUILD
+++ b/mingw-w64-cmake/PKGBUILD
@@ -13,10 +13,10 @@ url="https://www.cmake.org/"
 license=("MIT")
 makedepends=("${MINGW_PACKAGE_PREFIX}-gcc"
              "${MINGW_PACKAGE_PREFIX}-bzip2"
-             "${MINGW_PACKAGE_PREFIX}-ncurses"
-             "${MINGW_PACKAGE_PREFIX}-python-sphinx"
+             #"${MINGW_PACKAGE_PREFIX}-ncurses"
+             #"${MINGW_PACKAGE_PREFIX}-python-sphinx"
              "${MINGW_PACKAGE_PREFIX}-xz"
-             "${MINGW_PACKAGE_PREFIX}-zstd"
+             #"${MINGW_PACKAGE_PREFIX}-zstd"
              $([[ ${MINGW_PACKAGE_PREFIX} == *-clang-* ]] || \
                [[ ${MINGW_PACKAGE_PREFIX} == *-ucrt-* ]] || \
                echo "${MINGW_PACKAGE_PREFIX}-emacs" \
@@ -24,10 +24,10 @@ makedepends=("${MINGW_PACKAGE_PREFIX}-gcc"
 
 depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs"
          "${MINGW_PACKAGE_PREFIX}-pkg-config"
-         "${MINGW_PACKAGE_PREFIX}-curl"
+         #"${MINGW_PACKAGE_PREFIX}-curl"
          "${MINGW_PACKAGE_PREFIX}-expat"
-         "${MINGW_PACKAGE_PREFIX}-jsoncpp"
-         "${MINGW_PACKAGE_PREFIX}-libarchive"
+         #"${MINGW_PACKAGE_PREFIX}-jsoncpp"
+         #"${MINGW_PACKAGE_PREFIX}-libarchive"
          "${MINGW_PACKAGE_PREFIX}-libuv"
          "${MINGW_PACKAGE_PREFIX}-rhash"
          "${MINGW_PACKAGE_PREFIX}-zlib")
@@ -91,12 +91,15 @@ build() {
   MSYSTEM=MINGW FC=${MINGW_PREFIX}/bin/gfortran.exe \
     "${srcdir}"/${_realname}-${pkgver}/configure  \
     --prefix=${MINGW_PREFIX}                      \
-    --system-libs                                 \
     --parallel=${NUMBER_OF_PROCESSORS}            \
     --mandir=share/man                            \
     --docdir=share/doc/cmake                      \
-    --sphinx-man --sphinx-html                    \
-    --bootstrap-system-jsoncpp                    \
+    --system-expat                                \
+    --system-zlib                                 \
+    --system-bzip2                                \
+    --system-liblzma                              \
+    --system-librhash                             \
+    --system-libuv                                \
     --bootstrap-system-libuv                      \
     --bootstrap-system-librhash                   \
     ${_qtconfig[@]}
-- 
2.31.1.windows.1

