From e920991cdbef50f09440154975a570d111a5ccd8 Mon Sep 17 00:00:00 2001
From: Jeremy Drake <github@jdrake.com>
Date: Wed, 7 Apr 2021 16:19:30 -0700
Subject: [PATCH 03/11] zstd: use MSYS Makefiles instead of ninja for
 bootstrapping

---
 mingw-w64-zstd/PKGBUILD | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/mingw-w64-zstd/PKGBUILD b/mingw-w64-zstd/PKGBUILD
index 116a5e448..32196894a 100644
--- a/mingw-w64-zstd/PKGBUILD
+++ b/mingw-w64-zstd/PKGBUILD
@@ -11,7 +11,6 @@ mingw_arch=('mingw32' 'mingw64' 'ucrt64')
 url='https://facebook.github.io/zstd/'
 license=('BSD')
 makedepends=("${MINGW_PACKAGE_PREFIX}-gcc"
-             "${MINGW_PACKAGE_PREFIX}-ninja"
              "${MINGW_PACKAGE_PREFIX}-cmake")
 source=("https://github.com/facebook/zstd/releases/download/v${pkgver}/${_realname}-${pkgver}.tar.gz"
         zstd-1.4.0-fileio-mingw.patch)
@@ -36,7 +35,7 @@ build() {
 
   MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
     ${MINGW_PREFIX}/bin/cmake \
-      -GNinja \
+      -G"MSYS Makefiles" \
       -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
       "${extra_config[@]}" \
       -DZSTD_BUILD_CONTRIB=ON \
-- 
2.31.1.windows.1

