From 269e75b99c1dc2e02abf4b89de6e7e2cc48f1470 Mon Sep 17 00:00:00 2001
From: Jeremy Drake <github@jdrake.com>
Date: Tue, 19 Jan 2021 11:01:50 -0800
Subject: [PATCH 07/11] libpsl: use configure/make for bootstrapping

---
 mingw-w64-libpsl/PKGBUILD | 33 ++++++++++++++-------------------
 1 file changed, 14 insertions(+), 19 deletions(-)

diff --git a/mingw-w64-libpsl/PKGBUILD b/mingw-w64-libpsl/PKGBUILD
index c18899031..83c4f708a 100644
--- a/mingw-w64-libpsl/PKGBUILD
+++ b/mingw-w64-libpsl/PKGBUILD
@@ -14,38 +14,33 @@ depends=("${MINGW_PACKAGE_PREFIX}-libiconv"
          "${MINGW_PACKAGE_PREFIX}-libidn2"
          "${MINGW_PACKAGE_PREFIX}-libunistring"
          "${MINGW_PACKAGE_PREFIX}-gettext")
-makedepends=("${MINGW_PACKAGE_PREFIX}-libxslt"
-             "${MINGW_PACKAGE_PREFIX}-python"
-             "${MINGW_PACKAGE_PREFIX}-gettext"
-             "${MINGW_PACKAGE_PREFIX}-gtk-doc"
-             "${MINGW_PACKAGE_PREFIX}-meson"
-             "${MINGW_PACKAGE_PREFIX}-ninja")
+#makedepends=("${MINGW_PACKAGE_PREFIX}-libxslt"
+#             "${MINGW_PACKAGE_PREFIX}-python"
+#             "${MINGW_PACKAGE_PREFIX}-gettext"
+#             "${MINGW_PACKAGE_PREFIX}-gtk-doc"
+#             "${MINGW_PACKAGE_PREFIX}-meson"
+#             "${MINGW_PACKAGE_PREFIX}-ninja")
 source=("https://github.com/rockdaboot/libpsl/releases/download/${pkgver}/${_realname}-${pkgver}.tar.gz")
 sha256sums=('ac6ce1e1fbd4d0254c4ddb9d37f1fa99dec83619c1253328155206b896210d4c')
 
 build() {
+  export lt_cv_deplibs_check_method='pass_all'
   [[ -d "${srcdir}"/build-${CARCH} ]] && rm -rf "${srcdir}"/build-${CARCH}
   mkdir -p "${srcdir}"/build-${CARCH} && cd "${srcdir}"/build-${CARCH}
 
-  LDFLAGS+=" -liconv"
-  MSYS2_ARG_CONV_EXCL="--prefix=" \
-  meson \
+  ../${_realname}-${pkgver}/configure \
     --prefix="${MINGW_PREFIX}" \
-    --buildtype plain \
-    -Ddemos=true \
-    -Dwidgetry=true \
-    -Dgtk_doc=true \
-    -Druntime=libidn2 \
-    -Dbuiltin=libidn2 \
-    --default-library both \
-    ../${_realname}-${pkgver}
+    --enable-static \
+    --enable-shared \
+    --enable-runtime=libidn2 \
+    --enable-builtin=libidn2
 
-  ninja
+  make
 }
 
 package() {
   cd "${srcdir}"/build-${CARCH}
-  DESTDIR="${pkgdir}" ninja install
+  make install DESTDIR="${pkgdir}"
 
   install -Dm644 ${srcdir}/${_realname}-${pkgver}/COPYING ${pkgdir}${MINGW_PREFIX}/share/licenses/${_realname}/COPYING
 }
-- 
2.31.1.windows.1

