From e7e5ac89a477a7366c3a7203718edc0ae41f7d0f Mon Sep 17 00:00:00 2001
From: Jeremy Drake <github@jdrake.com>
Date: Fri, 8 Jan 2021 14:35:06 -0800
Subject: [PATCH 08/11] jemalloc: disable docbook for now

---
 mingw-w64-jemalloc/PKGBUILD | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/mingw-w64-jemalloc/PKGBUILD b/mingw-w64-jemalloc/PKGBUILD
index 5e1c7465c..89b430eb9 100644
--- a/mingw-w64-jemalloc/PKGBUILD
+++ b/mingw-w64-jemalloc/PKGBUILD
@@ -11,7 +11,7 @@ mingw_arch=('mingw32' 'mingw64' 'ucrt64')
 url="http://jemalloc.net/"
 license=('BSD')
 depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs")
-makedepends=("${MINGW_PACKAGE_PREFIX}-docbook-xsl")
+#makedepends=("${MINGW_PACKAGE_PREFIX}-docbook-xsl")
 options=(strip staticlibs !debug)
 source=("https://github.com/jemalloc/jemalloc/releases/download/${pkgver}/${_realname}-${pkgver}.tar.bz2"
         001-fix-library-extension.patch
@@ -41,13 +41,12 @@ build() {
   mkdir -p "${srcdir}"/build-${CARCH} && cd "${srcdir}"/build-${CARCH}
 
   CFLAGS+=" -std=gnu11"
-  _XMLROOT=$(ls -d ${MINGW_PREFIX}/share/xml/docbook/xsl-stylesheets* | tail -n1)
+  #_XMLROOT=$(ls -d ${MINGW_PREFIX}/share/xml/docbook/xsl-stylesheets* | tail -n1)
   ../${_realname}-${pkgver}/configure \
     --prefix=${MINGW_PREFIX} \
     --build=${MINGW_CHOST} \
     --host=${MINGW_CHOST} \
     --enable-autogen \
-    --with-xslroot=${_XMLROOT} \
     ${_btype}
 
   make
-- 
2.31.1.windows.1

