From 5bc720ef67956b8998aabaa7cdd68f8b23c61387 Mon Sep 17 00:00:00 2001
From: Jeremy Drake <github@jdrake.com>
Date: Sat, 23 Jan 2021 12:59:33 -0800
Subject: [PATCH 10/11] p11-kit: switch to msys2 gtk-doc to reduce mingw
 dependencies

---
 mingw-w64-p11-kit/PKGBUILD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mingw-w64-p11-kit/PKGBUILD b/mingw-w64-p11-kit/PKGBUILD
index 8c388e8ee..3d5f85dee 100644
--- a/mingw-w64-p11-kit/PKGBUILD
+++ b/mingw-w64-p11-kit/PKGBUILD
@@ -17,7 +17,7 @@ depends=("${MINGW_PACKAGE_PREFIX}-libtasn1"
          "${MINGW_PACKAGE_PREFIX}-gettext")
 makedepends=("${MINGW_PACKAGE_PREFIX}-gcc"
              "${MINGW_PACKAGE_PREFIX}-pkg-config"
-             "${MINGW_PACKAGE_PREFIX}-gtk-doc"
+             "gtk-doc"
              #"${MINGW_PACKAGE_PREFIX}-meson"
              #"${MINGW_PACKAGE_PREFIX}-ninja"
              )
-- 
2.31.1.windows.1

